depends/packages/boost.mk

```diff
@@ -33,18 +30,17 @@ endef
 
 # Fix unused variable in boost_process, can be removed after upgrading to 1.72
```

Comment:
Can drop this comment along with the `sed`.

```suggestion
```

Reply:
Dropped!